src/content/docs/release-notes/logs-release-notes/logs-000.mdx

```diff
@@ -0,0 +1,20 @@
+---
+subject: Logs
+releaseDate: '2021-07-31'
+version: 0.0.0
```

Comment:
Logs team talked about changing "version" to a date numbering system so we may change 0.0.0 to 210731 and rename this file